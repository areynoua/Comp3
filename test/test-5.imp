begin

Attention

(* Commentaire avec des ca		    ractères spécieaux et tr¢s „$ÏÜ‚ËÐÛÂÔ‚Î←ËÐŒ*îðëïüß)‘ ôûð‘€" *)
end
