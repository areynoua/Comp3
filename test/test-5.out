Unexpected symbol:
'end' with TOS: ':='
LL1Parser:
    token number: 2
    stack: [ ':=', '<Assign-Tail>', '<InstList-Tail>', 'end', '$' ]
    token: end	lexical unit: END
    line: 6, column: 0
    code: begin Attention end 

